openproblems/tasks/dimensional_reduction/datasets/__init__.py

```diff
@@ -1 +1,2 @@
-##
+from .brain_10x import load_10xbrain
```

Comment:
We removed this, right?

Reply:
Yes, thanks for the reminder. Fixed __init__.py